packages/frontend-shared/src/locales/en-US.json

```diff
@@ -647,6 +647,18 @@
     "notLoggedIn": "Not logged in",
     "notConnected": "Not connected",
     "noRuns": "No runs",
+    "emptyStates": {
+      "debugDirectlyInCypress": "Debug failed CI runs directly in Cypress",
+      "reviewRerunAndDebug": "Review, rerun, and debug failed CI test runs that are recorded to Cypress Cloud – all from within your local Cypress app.",
+      "connectToCypressCloud": "Connect to Cypress Cloud",
+      "notLoggedInTestMessage": "Connect to Cypress Cloud to locally debug failed CI test runs",
+      "noProjectTestMessage": "Connect a Cypress Cloud project to locally debug failed CI test runs",
+      "recordYourFirstRun": "Record your first test run to Cypress Cloud",
+      "almostThere": "You're almost there! Start recording your test runs to Cypress Cloud by running the command below in your local or CI terminal.",
+      "noRunsTestMessage": "Record your first test run to Cypress Cloud to locally debug failed CI test runs",
+      "gitRepositoryNotDetected": "Git repository not detected",
+      "ensureGitSetupCorrectly": "Cypress uses git to associate runs with your local state. Please ensure that version control is set up correctly."
```

Comment:
Should "git" be formatted differently when we render these messages? `git` or "git"?

Reply:
Good question... I think we should just capitalize it because it's a proper noun? 
> Git